static/js/components/UserPageAboutMeDialog.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import Button from 'react-mdl/lib/Button';
+
+import ProfileFormFields from '../util/ProfileFormFields';
+import type { Profile, SaveProfileFunc } from '../flow/profileTypes';
+import type { UIState } from '../reducers/ui';
+
+export default class UserPageAboutMeDialog extends ProfileFormFields {
+  props: {
+    ui:                           UIState,
+    profile:                      Profile,
+    saveProfile:                  SaveProfileFunc,
+    clearProfileEdit:             () => void,
+    setUserPageAboutMeDialogVisibility:  () => void,
+  };
+
+  closeAboutMeDialog: Function = (): void => {
+    const {
+      setUserPageAboutMeDialogVisibility,
+      clearProfileEdit,
+      profile: { username }
+    } = this.props;
+    setUserPageAboutMeDialogVisibility(false);
+    clearProfileEdit(username);
+  };
+
+  saveAboutMeInfo: Function = (): void => {
+    const { profile, ui, saveProfile } = this.props;
+    saveProfile(null, profile, ui).then(() => {
+      this.closeAboutMeDialog();
+    });
+  };
+
+  render () {
+    const { ui: { userPageAboutMeDialogVisibility } } = this.props;
+
+    const actions = [
+      <Button
+        type='button'
+        className='secondary-button cancel-button'
+        key='cancel'
+        onClick={this.closeAboutMeDialog}>
+        Cancel
+      </Button>,
+      <Button
+        type='button'
+        className='primary-button save-button'
+        key='save'
+        onClick={this.saveAboutMeInfo}>
+        Save
+      </Button>
+    ];
+
+    return (
+      <Dialog
+        title="About Me"
+        titleClassName="dialog-title"
+        contentClassName="dialog about-me-dialog"
+        className="about-me-dialog-wrapper"
+        open={userPageAboutMeDialogVisibility}
+        onRequestClose={this.closeAboutMeDialog}
+        actions={actions}
+        autoScrollBodyContent={true}>
+       {
+         this.boundTextField(
+           ["about_me"],
+           "",
```

Comment:
You are creating a text field without a `label`, which means that a screenreader can't determine what this text field is for. I had thought/hoped that the `hintText` field would be accessible, but it isn't -- I just verified with VoiceOver. Might be easiest to move the `hintText` to the `label`, or maybe duplicate them.